dcc-portal-ui/app/scripts/pathwayviewer/js/services.js

```diff
@@ -1082,58 +1125,33 @@ angular.module('icgc.pathwayviewer.directives.services', [])
           return newType;
         }
 
-
-        for(var i=0;i<types.length;i++){
-          x = i%2===0?marginLeft:marginLeft+100+10;
-          y = Math.floor(i/2)*40 + marginTop + 5*Math.floor(i/2);
-          var type = types[i];
-
-          if (type === overlappedNodeText) {
-            x = marginLeft;
-            y = Math.floor((i+ 1)/2)*40 + marginTop + 5*Math.floor((i+1)/2);
-          }
-
-
-          var node = {
-            position:{x:x,y:y},
-            size:{width:90,height:30},
-            type: _getNodeType(type),
-            id: _getIDForType(type),
-            crossed:type===failedText?true:false,
-            lof:type===lofText?true:false,
-            grayed: false,
-            reactomeId: _getIDForType(type),
-            text:{content:type,position:{x:x,y:y}}
-          };
-          //console.log(node);
-          if (type === overlappedNodeText) {
-            node.size.height += 10;
-          }
-
-          nodes.push(node);
-
-          if (type === mutatedNodeText) {
+        function addMutatedNodeComment() {
             // Add extra comment for mutated gene node to show what the value in the corner means
             svg.append('foreignObject').attr({
-              x: marginLeft+90,
+              x: marginLeft+100,
               y: y - 15,
               width:100,
               height:35,
               'fill':'none'
             }).append('xhtml:body')
               .attr('class','RenderableNodeText')
               .html('<table class="RenderableNodeTextCell"><tr><td valign="middle">'+
-                    '&larr; # ICGC Mutations'+'</td></tr></table>');
-
-
-          }
-
-
+                    '<--- # ICGC Mutations'+'</td></tr></table>');
         }
 
-
-
-        return nodes;
+        function addDruggableNodeComment() {
+            // Add extra comment for druggable gene node to show what the value in the corner means
+            svg.append('foreignObject').attr({
+              x: 5,
+              y: y - 22,
+              width:275,
+              height:35,
+              'fill':'none'
+            }).append('xhtml:body')
+              .attr('class','RenderableNodeText')
```

Comment:
Where is this class defined? Perhaps we should use a more standard naming scheme here?